### DISCUSSION TITLE
after 6 layer can't add html tag

### DISCUSSION THREAD

**Discussion by bigbear1024** [markmap/markmap#240] [Q&A] [answered]
Hi,
When I add html tag like `<p>` to 7th layer, and export to html, added tag text didn't show up.
how to fix it?
thank you!
![1712500972399](https://github.com/markmap/markmap/assets/50311460/0de78db7-d50b-4e8d-811b-d69679849ada)

**Comment by gera2ld** [comment]
https://markmap.js.org/docs/faq

**Reply by bigbear1024** [reply]
I used nested list to 7th level and my problem is html tag didn’t work.
When I add the tag, the text will not display.

**Reply by gera2ld** [reply, marked as answer]
It seems to be a bug of remarkablejs. I will switch to markdown-it in the next version. You can try it now in the [REPL](https://markmap.js.org/repl).
